modules/server/src/network/resolvers/aggregations.ts

```diff
@@ -33,77 +35,18 @@ const fetchData = async (query: NetworkQuery) => {
 			return failure(CONNECTION_STATUS.ERROR, `Request cancelled: ${url}`);
 		}
 
-		const responseStatus = error.response.status;
-		if (responseStatus === 404) {
-			console.error(`Network failure: ${url}`);
-			return failure(CONNECTION_STATUS.ERROR, `Network failure: ${url}`);
-		} else {
-			return failure(CONNECTION_STATUS.ERROR, `Unknown error`);
+		if (axios.isAxiosError(error)) {
+			if (error.code === 'ECONNREFUSED') {
+				console.error(`Network failure: ${url}`);
+				return failure(CONNECTION_STATUS.ERROR, `Network failure: ${url}`);
+			}
 		}
```

Comment:
catches error is server is unavailable
nested for now - would like to check if there are other codes need to be covered